scripts/log_wrapper.sh

```diff
@@ -15,5 +15,5 @@ if [ $sessionCreated -eq 0 ]; then
     lttng destroy msquic
 
     babeltrace --names all $dirname/data > $dirname/quic.babel.txt
-    ./submodules/clog/src/clog2text/clog2text_lttng/bin/Release/net6.0/publish/clog2text_lttng -i $dirname/quic.babel.txt -s ./src/manifest/clog.sidecar -o $dirname/quic.log --showTimestamp --showCpuInfo
+    ./submodules/clog/src/clog2text/clog2text_lttng/bin/Debug/net6.0/publish/clog2text_lttng -i $dirname/quic.babel.txt -s ./src/manifest/clog.sidecar -o $dirname/quic.log --showTimestamp --showCpuInfo
```

Comment:
We shouldn't be using debug

Reply:
`pwsh ./scripts/prepare-machine.ps1` generates this Debug build automatically though

Maybe this

```pwsh
    dotnet build (Join-Path $RootDir submodules clog)
```